pages/sign-in/index.page.tsx

```diff
@@ -0,0 +1,57 @@
+import Image from "next/image";
+import { useRouter } from "next/router";
+
+import classNames from "classnames/bind";
+
+import styles from "./index.module.scss";
+
+const cn = classNames.bind(styles);
+
+const KAKAO_REDIRECT_URI = "http://localhost:3000/sign-in/kakao";
+const KAKAO_AUTH_URL = `https://kauth.kakao.com/oauth/authorize?client_id=${process.env.NEXT_PUBLIC_REST_API_KEY}&redirect_uri=${KAKAO_REDIRECT_URI}&response_type=code`;
+
+export default function SignIn() {
+  const router = useRouter();
+
+  const handleKakaoButtonClick = () => {
+    router.push(KAKAO_AUTH_URL);
+  };
+
+  return (
+    <>
+      <header className={cn("header")}>
+        <Image src="/icons/tickita-logo.svg" alt="logo" width={60} height={22} />
+      </header>
+      <main>
+        <div className={cn("container")}>
+          <div className={cn(["tickita-logo"])}>
```

Comment:
```suggestion
        <div className={cn("container")}>
          <div className={cn(["tickita-logo"])}>
```
위에는 그냥 문자열인데 아래는 배열로 한번 감싸신 이유가 있으신가요??